src/stores/RenderConfigStore.ts

```diff
@@ -277,6 +304,42 @@ export class RenderConfigStore {
         this.updateSiblings();
     };
 
+    @action setBias = (bias: number) => {
+        this.bias = bias;
+    };
+
+    @action resetBias = () => {
+        this.bias = 0;
+    };
+
+    @action setContrast = (contrast: number) => {
+        this.contrast = contrast;
+    };
+
+    @action resetContrast = () => {
+        this.contrast = 1;
+    };
+
+    @action setSmoothedBias = (smoothedBias: number) => {
```

Comment:
again, I think we should just use the same bias/contrast variables here. When we switch modes, we can automatically reset them to the correct defaults

Reply:
Combined the parameters.